Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2.7
+
+RUN mkdir -p /opt/ecs-rollover
+COPY requirements.txt /opt/ecs-rollover
+RUN pip install -r /opt/ecs-rollover/requirements.txt
+
+COPY src/__init__.py /opt/ecs-rollover/
```

Comment:
It may be fixed, but the last time I wild carded a dir, docker build didn't always pick up changes to the files when I made them